operator/spec.go

```diff
@@ -20,13 +20,6 @@ type Operator interface {
 	// and return information about Flannel bridges, how they should look like on
 	// a server host.
 	GetDesiredState(obj interface{}) (interface{}, error)
-	// GetEmptyState is only to return the specific zero value the operator
-	// expects when reconciling delete operations. So this returns the desired
-	// state for delete operations. On create and delete events the operator might
-	// need to reconcile resources by removing them. GetDeleteState will receive
-	// as desiredState. This is to align to the general concept of reconciliation
-	// regardless creation or deletion events.
-	GetEmptyState() interface{}
```

Comment:
As stated above this is not needed anymore. `GetDeleteState` receives the real desired state, which feels way better than this.